frontend/src/util/Hook.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import { Player, Submission } from '../api/Game';
+
+// eslint-disable-next-line import/prefer-default-export
+export const useBestSubmission = (player?: Player) => {
```

Comment:
I made it non-default cause I figured since there will most likely be more functions joining it, making it default would mean we'd have to refactor all usages across files (I think there are 3) of `import X from Y` to `import { X } from Y`. But it's a small tradeoff between that and having to disable the ESLint rule, so I'm fine changing it to default if you'd prefer that solution. 

Also, I made a separate file for this since I figured we'll most likely be introducing more [custom hooks](https://reactjs.org/docs/hooks-custom.html) in the future, so this might be a good place to store them.

Reply:
Got it sounds good - I'm ok leaving this as-is, as long as we remember to remove that comment once we add more functions.